src/device.c

```diff
@@ -19,7 +19,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#include "device.h"
+#include "../include/device.h"
```

Comment:
Agreed, using relative paths in "#include" statements should be avoided. If it is an issue, it should be fixed in CMakeLists by include_directories directive.